<?xml version="1.0"?>
<launch>
    <!--argument-->
    <arg name="low_lines" default="6" />
    <arg name="z_threshold" default="0.03" />
    <arg name="distance_threshold" default="0.2" />
    <arg name="map_tf" default="velo_middle" />
    <arg name="lowest_segment_point_num" default="7" />

<!--    <node pkg="ground_detect" type="lidar_line_analyse" name="lidar_line_analyse_node" output="screen">-->
<!--        <remap from="/lidar/vlp32_middle/PointCloud2" to="/lidar/vlp32_middle/PointCloud2" />-->
<!--        <param name="low_lines" value="$(arg low_lines)" />-->
<!--&lt;!&ndash;        <param name="z_threshold" value="$(arg z_threshold)" />&ndash;&gt;-->
<!--&lt;!&ndash;        <param name="distance_threshold" value="$(arg distance_threshold)" />&ndash;&gt;-->
<!--    </node>-->

    <node pkg="ground_detect" type="feature_based_select" name="feature_based_select_node" output="screen">
        <param name="low_lines" value="$(arg low_lines)" />
        <param name="distance_threshold" value="$(arg distance_threshold)" />
        <param name="lowest_segment_point_num" value="$(arg lowest_segment_point_num)" />
    </node>

<!--    &lt;!&ndash;rviz&ndash;&gt;-->
<!--    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ground_detect)/rviz/result.rviz -f $(arg map_tf)" />-->
</launch>
